app/services/calculators/multiple_employments_calculator.rb

```diff
@@ -12,7 +12,11 @@ def call(submission_date)
     private
 
       DummyEmploymentFigures = Data.define(:monthly_gross_income, :monthly_benefits_in_kind, :monthly_tax, :monthly_national_insurance) do
-        def actively_working?
+        def entitles_employment_allowance?
+          false
```

Comment:
Think we need for this rather than reducing the coverage numbers. This value IMHO should be true not false